.travis.yml

```diff
@@ -69,7 +69,7 @@ matrix:
           - ubuntu-toolchain-r-test
           packages:
           - binutils-dev
-          - g++-5
+          - g++-8
```

Comment:
You'll have to fix https://github.com/symengine/symengine/blob/master/bin/install_travis.sh#L35 as well